share/pvss/pvss.go

```diff
@@ -0,0 +1,172 @@
+// Package pvss implements public verifiable secret sharing as introduced in
+// "A Simple Publicly Verifiable Secret Sharing Scheme and its Application to
+// Electronic Voting" by Berry Schoenmakers. In comparison to regular verifiable
+// secret sharing schemes, PVSS enables any third party to verify shares
+// distributed by a dealer using zero-knowledge proofs.
+package pvss
```

Comment:
Reading again this PR after three weeks make me realize I still don't know how it would be used. I know it's boring but it'd be very helpful to have a good section explaining how to use this library and what to do with the shares / encrypted shares etc. Just the "usage" description.